vllm/envs.py

```diff
@@ -128,6 +128,7 @@
     VLLM_USE_DEEP_GEMM: bool = False
     VLLM_SKIP_DEEP_GEMM_WARMUP: bool = False
     VLLM_USE_FLASHINFER_MOE_FP8: bool = False
+    VLLM_FLASHINFER_MOE_FP8_BACKEND: str = "flashinfer"
```

Comment:
should we call it `trtllm-gen` vs `cutlass` or `latency` vs `throughput` like the FP4 MoE PR? https://github.com/vllm-project/vllm/pull/21331